modules/custom/d_p/src/Plugin/Field/FieldWidget/ButtonLinkWidget.php

```diff
@@ -39,7 +39,10 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
       '#default_value' => $options['type'] ?? ButtonTypeEnum::Primary->value,
     ];
 
-    return $element + parent::formElement($items, $delta, $element, $form, $form_state);
+    $parent_elements = parent::formElement($items, $delta, $element, $form, $form_state);
+    $parent_elements["title"]["#required"] = TRUE;
```

Comment:
Let's use a single quotation mark for consistency.

Reply:
Fixed!